tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -33,8 +48,34 @@ exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
     }
   }
 
-  function writeToFile(filename, json) {
-    const fullPath = `${mobileStaticPath}/mobile/${filename}.json`;
+  function getBlockDescription(superBlockKey, blockKey) {
+    try {
+      const intros = JSON.parse(fs.readFileSync(blockIntroPath));
+
+      const introValues = intros[superBlockKey]['blocks'][blockKey]['intro'];
+
+      return introValues.join('');
+    } catch (e) {
+      return '';
+    }
```

Comment:
```suggestion
    const intros = JSON.parse(fs.readFileSync(blockIntroPath));

    return intros[superBlockKey]['blocks'][blockKey]['intro'];
```
Couple of things. 

The try catch seems counter productive as we want to this script to fail if something unexpected happens.

Each element of the intro array is a paragraph, so you'll probably need that information in the mobile app.